src/components/SearchField.tsx

```diff
@@ -203,13 +214,16 @@ const SearchField: React.FC = () => {
               </label>
               <select value={genreToSearch} onChange={selectGenreHandler}>
                 <option value="none">Select genre</option>
-                {Object.values(Genres).map((genre, i) => (
-                  <option value={genre} key={i}>
-                    {genre}
+                {Object.entries(Genres).map(([key, value]) => (
+                  <option value={key} key={key}>
+                    {value}
                   </option>
                 ))}
               </select>
             </div>
+            <div className="search-field-message-wrapper">
+              {isDataValid && <p className="search-field-message">You are trying to choose invalid date range</p>}
```

Comment:
Valid means correct, invalid - incorrect. So if isDataValid = true we are showing error message?

Reply:
Gave an appropriate name for a state